core.js

```diff
@@ -149,7 +149,26 @@ class SiteMapper {
             }
             let priority = '';
             let changefreq = '';
-            if (this.pagesConfig && this.pagesConfig[pagePath.toLowerCase()]) {
+            if (!this.pagesConfig) {
+                return {
+                    pagePath,
+                    outputPath,
+                    priority,
+                    changefreq
+                };
+            }
+            // 1. Generic wildcard configs go first
+            Object.entries(this.pagesConfig).forEach(([key, val]) => {
+                if (key.includes("*")) {
```

Comment:
Happy to come up with a better strategy to enable regex support on a particular page config?

Reply:
interesting, I will check it tomorrow